packages/components/src/components/Alert/Alert.tsx

```diff
@@ -33,13 +32,23 @@ export type AlertProps = {
   primaryButton?: ButtonProps
   /** Optional secondary action button */
   secondaryButton?: ButtonProps
-  /** Optional boolean for determining when to focus on error alert boxes (e.g. onSaveClicked). */
-  // focusOnError?: boolean
-  /** Optional ref for the parent scroll view. Used for scrolling to error alert boxes. */
-  // scrollViewRef?: RefObject<ScrollView>
-  /** optional testID */
+  /** Optional testID */
   testId?: string
-}
+} & (
+  | {
+      /** True to make the Alert expandable */
+      expandable: true
+      /** Header text. Required if expandable. Optional if not */
+      header: string
+      /** True if Alert should start expanded. Defaults to false */
+      initializeExpanded?: boolean
+    }
+  | {
+      expandable?: false
```

Comment:
Didn't check, but may also need doc on expandable/header here to correctly show helper text when leveraging the component since by default expandable would be false.

Reply:
Good call. Added.